suite_cam6.xml

```diff
@@ -0,0 +1,360 @@
+      <scheme>convtran</scheme>                 <!-- Actual ZM convective transport scheme, for constituents in cnst_is_convtran1 -->
+      <scheme>convtran_cam_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <!-- **************************** -->
+      <scheme>check_energy_chng</scheme>        <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_tracers_chng</scheme>       <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <scheme>sslt_rebin_adv</scheme>           <!-- Only matters if certain sea salt species are present in chemistry package-->
+      <subcycle loop="cloud_macmic_num_steps">  <!-- Cloud macrophysics-microphysics loop -->
+        <!-- CLUBB turbulence, convection, and macrophysics -->
+        <!-- **************************** -->
+        <!--This section will likely require additional clean-up,
+            as many of these routines are simply unit or vertical
+            grid conversions, or the re-naming and re-setting
+            of host model inputs.  Thus they can likely be
+            managed by the framework itself, and won't need
+            their own scheme calls.-->
+        <scheme>set_dry_to_wet</scheme>               <!-- Convert mixing ratios from dry air to air + water vapor -->
+        <scheme>grid_size</scheme>                    <!-- Calculate grid-cell size (not sure if this will be managed by registry/interface?) -->
+        <scheme>clubb_dtime_calc</scheme>             <!-- Calculate CLUBB time-step as function of host model time-step -->
+        <scheme>exner_clubb</scheme>                  <!-- Calculate Exner function using CLUBB definition (should be manageable via the registry/meta file) -->
+        <scheme>tropopause_findChemTrop</scheme>      <!-- Calculate tropopause height -->
+        <scheme>clubb_vertical_grid_inv</scheme>      <!-- Change order of vertical levels (should be manageable by registry/meta file) -->
+        <scheme>clubb_calc_ustar</scheme>             <!-- Only matters for single-column mode -->
+        <scheme>setup_grid_heights_api</scheme>       <!-- Re-calculate CLUBB grid heights -->
+        <scheme>setup_parameters_api</scheme>         <!-- Re-calculate grid-dependent CLUBB parameters -->
+        <scheme>zt2zm_convert</scheme>                <!-- Transition variables on thermodynamic levels to momentum levels (manageable by registry/meta file?) -->
+        <scheme>clubb_var_vert_inv</scheme>           <!-- Change order of vertical levels for CLUBB input variables (should be manageable via the registry/meta file) -->
+        <subcycle loop="clubb_nadv">                  <!-- CLUBB sub-step loop (number of iterations should be calculated in the "clubb_dtime_calc" scheme) -->
+          <scheme>stats_begin_timestep_api</scheme>   <!-- Only matters if "l_stats" is True -->
```

Comment:
Should this be a timestep_init routine?

Reply:
Given that this scheme is run for every CLUBB sub-step (as opposed to the host model time step), I wasn't sure if it was an official "timestep_init" routine, or if it had to be treated as a regular physics scheme?